includes/Meta_Boxes.php

```diff
@@ -37,7 +37,7 @@ class Meta_Boxes {
 	 *
 	 * @var string[]
 	 */
-	const LOCATIONS = [ 'normal', 'advanced' ];
+	const LOCATIONS = [ 'normal', 'advanced', 'side' ];
```

Comment:
Where do sidebar based metaboxes show up?

Reply:
Same location in the editor (at the bottom)